recipes-tag/event-model/meta.yaml

```
@@ -9,13 +9,13 @@ source:
   git_rev: v{{ version }}
 
 build:
-  number: 1
+  number: 2
   script: python setup.py install --single-version-externally-managed --record=record.txt
   skip: True  # [py2k]
```

```suggestion
  skip: True  # [py>=3.6]
```
From https://docs.conda.io/projects/conda-build/en/latest/source/resources/define-metadata.html#preprocessing-selectors

---

Thanks for the link! I think the condition should be opposite. See my comment below.